tests/migration_test.go

```diff
@@ -697,18 +697,14 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 				}, 240*time.Second, 1*time.Second).Should(BeTrue())
 			})
 		})
-		Context("with a shared ISCSI Filesystem PVC", func() {
+		Context("with a shared ISCSI Filesystem PVC (using ISCSI IPv4 address)", func() {
```

Comment:
Didn't find a better place to comment on the commit message. There is no single stack ipv6 lane. We don't support it anymore. We remove the skip because we changed the test to be ipv4 only not because now it will work on single stack ip lane.

Reply:
Done